### ISSUE TITLE
dbeaver@22.3.4: hash check failed

### ISSUE THREAD

**Issue by adrianocorrea** [ScoopInstaller/Extras#10449]
When updating `dbeaver`, one gets the following error: 

```sh
Checking hash of dbeaver-ce-22.3.4-win32.win32.x86_64.zip ... ERROR Hash check failed!
App:         extras/dbeaver
URL:         https://download.dbeaver.com/community/22.3.4/dbeaver-ce-22.3.4-win32.win32.x86_64.zip
First bytes: 50 4B 03 04 0A 00 00 08
Expected:    1d5bf1735002f29b8ff5dfcd503389e9c56ded72388c8b477ded27b69a4b420e
Actual:      5090f77478ff652770652f847d1168fe71c04375e194209ea793e383d40e6f98
```

I have manually downloaded DBeaver community edition, and the expected hash is correct. It seems that the mirror being used is retrieving a modified version of the software. 

I suggest replacing the current download URL with [DBever GitHub's official mirror](https://github.com/dbeaver/dbeaver/releases) to reduce the chance of Scoop retrieving malicious code by mistake.

**Comment by adrianocorrea** [comment]
Yes, my scoop is up to date. The problem is either with the hash or with the zip file scoop intended to use.
 
You can reproduce the error by downloading it from 2 different locations and comparing the hashs: 
* https://github.com/dbeaver/dbeaver/releases/download/22.3.4/dbeaver-ce-22.3.4-win32.win32.x86_64.zip (expected)
* https://dbeaver.io/files/dbeaver-ce-latest-win32.win32.x86_64.zip (actual)

These two locations are official DBeaver repositories.

**Comment by adrianocorrea** [comment]
Fixed by commit 0e7e52f
